capstone/src/main/java/com/step902020/capstone/Individual.java

```diff
@@ -111,10 +111,18 @@ public void deleteSavedEvents(long event) {
     savedEvents.remove(event);
   }
 
+  /**
+   * add a new organization to the list
+   * @param organization organization to be added
+   */
   public void addSavedOrganizations(long organization) {
```

Comment:
organization ID?

Reply:
This was before the change to use references. After the merge this will be taking in an Organization object